### ISSUE TITLE
Release date in ALMA query not correct

### ISSUE THREAD

**Issue by nbrunett** [astropy/astroquery#1340]
Hi

If I run

```python
from astroquery.alma import Alma
result = Alma.query_object('30 Doradus')
print(result['Release date'])
```

then I get

```
Release date
------------
           2
           2
           2
           2
           2
           2
           2
           2
           2
           2
```

instead of dates. The web form through a browser is showing dates as expected. My attempt at giving the "version" I'm using:
help(astroquery) gives
__githash__ = 'b7d2a0f102292ff85381b95582b9441ad288ceb2'
VERSION 0.3.10.dev0

**Comment by keflavich** [comment]
I can reproduce this.  I suspect there's an upstream change/error that caused this recently.  I'll investigate when I get a chance, but I welcome anyone else's effort to figure this out!

**Comment by nbrunett** [comment]
A bit of digging seems to indicate the Release date definition at the top of the XML table is different from all the rest of the fields (except for the Mosaic field which is actually showing the same problem, see below). What I see is

```xml
<FIELD name="Array" datatype="char" unit="" xtype="adql:VARCHAR" arraysize="*" />
<FIELD name="Mosaic" />
<FIELD name="Integration" datatype="double" ID="1213" ucd="time.duration;obs.exposure" unit="s" utype="Char.TimeAxis.Coverage.Support.Extent" xtype="adql:DOUBLE">
  <DESCRIPTION>Aggregated integration time for the field in the ASDM.</DESCRIPTION>
</FIELD>
<FIELD name="Release date" />
```

where there is no metadata associated with the Release date and Mosaic fields (beyond their names) while other fields have types specified. When I look at the raw XML response in _parse_result within astroquery/alma/core.py it has the full release date (e.g. 2017-01-01). After the response string has been parsed with votable.parse, the date is truncated to the first character like in my original report.

My lack of experience with XML/VOTable parsing means I can't say if this is an issue with poorly formatted XML or the votable parser having a hiccup, without digging some more. I don't have an example archive response string from before this problem arose either, unfortunately. I thought it would be worth reporting on what I found so far.

As for the Mosaic field having the same problem, that is supposed to say "mosaic" or be an empty string. Adding to my example from my first post

```python
from astroquery.alma import Alma
result = Alma.query_object('30 Doradus')
print(result['Mosaic'])
```

gives

```
Mosaic
------
     m
     m
      
     m
      
      
      
      
      
      

```

**Comment by keflavich** [comment]
Thanks for digging.  I can't deal with this right now, but the information will help when I can get to it.

**Comment by andamian** [comment]
Ran into the same issue.

This seems to be a problem related to the alma service not returning complete information. I can confirm that when the type and size of the fields are provided, the astroquery.alma modules works. Opened a ticket with almascience: https://help.almascience.org/index.php?/na/Tickets/Ticket/View/14715

**Comment by keflavich** [comment]
This is an upstream issue as @andamian noted.  I'll add a (very sad) hack to fix it in the meantime

**Comment by keflavich** [comment]
See #1369

**Comment by bsipocz** [comment]
This has been fixed in #1369
